cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage {
+  private addressText: string;
+  private passwordText: string;
+  private loginBtn: string;
+
+  constructor() {
+    this.addressText = "#email";
+    this.passwordText = "#passwd";
+    this.loginBtn = "button#SubmitLogin.button.btn.btn-default.button-medium";
+  }
+
+  public writeAddress(): void {
+    cy.get(this.addressText).type("aperdomobo@gmail.com");
+  }
+
+  public writePassword(): void {
+    cy.get(this.passwordText).type("WorkshopProtractor");
+  }
+
+  public login(): void {
+    cy.get(this.loginBtn).click();
+  }
+}
+
+export {LoginPage};
+
```

Comment:
delete this line, just leave one empty line at the end of the file

Reply:
resolved